src/Game.js

```diff
@@ -0,0 +1,125 @@
+class Frame {
+  constructor () {
+    this._rolls = [];
+    this._bonus_score = 0;
+  }
+
+  isStrike() {
+    return (this._rolls[0] == 10);
+  }
+
+  isSpare() {
+    return (this._rolls[0] + this._rolls[1]== 10);
+  }
+
+  isFrameComplete() {
+    if (this.isStrike()) { return true; }
```

Comment:
This may be enough to work, but it feels like also checking that there's one roll would be good? Happy to get pushback on this.